src/data/multiset/powerset.lean

```diff
@@ -253,4 +254,18 @@ begin
   { cases n; simp [ih, map_comp_cons], },
 end
 
+lemma disjoint_powerset_len (s : multiset α) {i j : ℕ} (h : i ≠ j) :
+  multiset.disjoint (s.powerset_len i) (s.powerset_len j) :=
+λ x hi hj, h (eq.trans (multiset.mem_powerset_len.mp hi).right.symm
+  (multiset.mem_powerset_len.mp hj).right)
+
+lemma bind_powerset_len {α : Type*} (S : multiset α) :
```

Comment:
This stuff was all in the preliminary PR, right? Did something go wrong with the merge?

Reply:
I think the merge was just never done. I've pushed a merge commit just to give bors a head start. The diff looks clean now.

Reply:
Thanks. I forgot to do the merge earlier.